roles/pulp-webserver/defaults/main.yml

```diff
@@ -4,3 +4,4 @@ pulp_content_host: 127.0.0.1
 pulp_content_port: 24816
 pulp_api_host: 127.0.0.1
 pulp_api_port: 24817
+pulp_webserver_manage_firewalld: True
```

Comment:
#160 uses the name `pulp_configure_firewalld` here. I would take even more suggestions.

Reply:
I started reviewing https://github.com/pulp/ansible-pulp/pull/160 . I suggested an alternative implementation.

Reply:
Also, once you do change it, you need to update the README.md.

Reply:
https://github.com/pulp/ansible-pulp/pull/160 was merged with the alternative implementation.